pkg/limayaml/defaults.go

```diff
@@ -299,6 +299,7 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	for _, mount := range append(append(d.Mounts, y.Mounts...), o.Mounts...) {
 		if i, ok := location[mount.Location]; ok {
 			mounts[i].Writable = mount.Writable
+			mounts[i].SSHFS = mount.SSHFS
```

Comment:
This is unfortunately not enough. Both `mount.Writable` and `mount.SSHFS.ForwardSymlinks` need to become pointers to booleans, so they can be independently defaulted/overridden. This should also be added to the `defaults_test.go` to make sure it works.

Not making `mount.Writable` a pointer to start with was a mistake already, but it becomes more obvious now that we have multiple properties on a mount.

Reply:
@jandubois, I just pushed a change changing to *bool. I can make the updates for Mount#Writable, but I figured that probably warrants its own PR.

Reply:
@jandubois, I just pushed a change changing to *bool. I can make the updates for Mount#Writable, but I figured that probably warrants its own PR.